### ISSUE TITLE
[Bug]: Nushell 0.92 breaks Atuin script

### ISSUE THREAD

**Issue by brizzbuzz** [atuinsh/atuin#1934]
### What did you expect to happen?

Hello :wave: nushell pushed some breaking changes in 0.92 that changes how `run-external` works, and thus breaks the existing atuin nushell script https://www.nushell.sh/blog/2024-04-02-nushell_0_92_0.html#deprecated-commands-toc

### What happened?

```shell
> Error:   × Deprecated flag                                                                                                                          04/08/2024 09:29:17 PM PM
   ╭─[entry #1:2:48]
 1 │ # c6fd19fa-f67d-4dae-bb81-1b9fdff43783
 2 │ commandline edit (ATUIN_LOG=error run-external --redirect-stderr atuin search "--shell-up-key-binding" "--interactive" "--" (commandline) | complete | $in.stderr | str substring ..-1)
   ·                                                ─────────────────────────────────────────────┬────────────────────────────────────────────
   ·                                                                                             ╰── `--redirect-stderr` is deprecated
   ╰────
  help: use the `e>|` stderr pipe redirection instead
```

### Atuin doctor output

```yaml
nothing is wrong w/ atuin :)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by arcuru** [comment]
Addressed in #1913?

**Comment by brizzbuzz** [comment]
womp womp I was looking at that script in the atuin repo on my local and never pulled the updated code... yea that should do it :)
